### PR TITLE
refactor(index): cleanup index.ts

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
cc @Foxandxss

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,31 +1,35 @@
-import {NgbAccordion, NgbPanel} from './accordion/accordion';
+import {NGB_ACCORDION_DIRECTIVES} from './accordion/accordion';
+import {NGB_ALERT_DIRECTIVES, NgbAlert} from './alert/alert';
```
Comment (Foxandxss) [inline src/index.ts]: extra NgbAlert

**Comment by Foxandxss** [comment]
Seems fine. I am confused in why you need to import some stuff like `NgbAlert` but not import the accordion parts in the previous line.

**Comment by pkozlowski-opensource** [comment]
> I am confused in why you need to import some stuff like NgbAlert but not import the accordion parts in the previous line.

we need this to export `NGB_PRECOMPILE`. This is needed so people can put this array in the `precompile` attribute of a component where they want to use dynamic stuff (currently alerts, tooltips, popovers but more in the future).

**Comment by Foxandxss** [comment]
Oh, missed that part. LGTM

**Pull request closed without merging**